qipr_approver/approver/urls.py

```diff
@@ -4,7 +4,7 @@
 
 urlpatterns = [
     url(r'^$', views.index, name='index'),
-    url(r'^shib/$', views.fake_shib, name='shib'),
+    url(r'^shib/$', views.shib_login, name='shib'),
```

Comment:
I avoided that with the SHIB_ENABLED setting which still should do all the same things that fakeshib did without the need of shib stuff